db/models/getApi.model.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../connection");
+const fs = require('fs/promises');
+
+exports.findApi = () => {
+    return fs.readFile(`${__dirname}/../../endpoints.json`, 'utf-8').then( (data) => {
```

Comment:
There is no need to read a json file which will convert it to a string, to then convert that string to a json file to send to the client...just send the json